test/BodyParams/BodyParamsMiddlewareTest.php

```diff
@@ -181,13 +188,43 @@ public function testThrowsMalformedRequestBodyExceptionWhenRequestBodyIsNotValid
         $this->expectExceptionMessage($expectedException->getMessage());
         $this->expectExceptionCode($expectedException->getCode());
 
-        $middleware($serverRequest, new Response(),
-            function ($request, $response) use (&$triggered) {
-                $triggered = true;
-                return $response;
-            }
+        $middleware->process(
+            $serverRequest,
+            $this->fakeDelegateNeverTriggered(
+                function (ServerRequestInterface $request) use (&$triggered) {
+                    $triggered = true;
+
+                    return new Response();
+                }
+            )
         );
 
         $this->assertFalse($triggered, 'Next should not have been triggered!');
     }
+
+    private function fakeDelegate(callable $callback)
+    {
+        $delegate = $this->createMock(DelegateInterface::class);
```

Comment:
We use `prophesize()` elsewhere within this class for creating mocks; any particular reason for using the PHPUnit built-ins instead?